docs/docs/examples/guides/ur-sim-guide.md

```diff
@@ -0,0 +1,146 @@
+---
+sidebar_position: 7
+title: URSim Guide
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import ursimActivate from './assets/ursim-activate.gif'
+import ursimHardwareParams from './assets/ursimHarwareParams.png'
+
+# URSim guide
+
+URSim is a software-in-the-loop simulation tool that allows to emulate the behavior of the real Universal Robots
+controller on a computer. This environment can be utilized to become familiar with the robot interface, as well as to
+create and run programs. Please note that compared to the real robot, there are some limitations to the simulator.
+Especially the force mode will be limited in use, since there is no physics engine to simulate valid force values from
+the sensor. Other functionality that is not present includes:
+
+- Emergency stop button
+- Input IO state
+- Self or surrounding object collisions
+
+For more information, please refer to the
+[installation page](https://www.universal-robots.com/download/software-ur-series/simulator-linux/offline-simulator-ur-series-e-series-ur-sim-for-linux-5220/).
```

Comment:
```suggestion
For more information, please refer to URs
[download page](https://www.universal-robots.com/download/software-ur-series/simulator-linux/offline-simulator-ur-series-e-series-ur-sim-for-linux-5220/).
```

Reply:
"UR's download page" or "the UR download page"